src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+* Addressed TypeDescriptor.GetProperties warning by removing it.
```

Comment:
It could be unclear to the reader what does this mean (and why would they care about this).

Reply:
updated changelog and title